Healthy/Classes/Modules/Login/LoginViewController.swift

```diff
@@ -57,6 +66,68 @@ private extension LoginViewController {
     }
 }
 
+// MARK: bind to text fields changes
```

Comment:
```suggestion
// MARK: TextFields changes
```
Sentence should start with a capital letter.